cmake/DyninstConfig.cmake.in

```diff
@@ -28,5 +28,4 @@ endforeach()
 check_required_components(Dyninst)
 
 # -- DO NOT USE -- This is for legacy purposes only
-set_and_check(DYNINST_INCLUDE_DIR
-              "${CMAKE_CURRENT_LIST_DIR}/../../../@DYNINST_INSTALL_INCLUDEDIR@")
+set_and_check(DYNINST_INCLUDE_DIR "@PACKAGE_DYNINST_INSTALL_INCLUDEDIR@")
```

Comment:
Is this from GNUInstallDirs?

Reply:
This is from [CMakePackageConfigHelpers](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html). In detail:

> The variables `<var1>` to `<varN>` given as `PATH_VARS` are the variables which contain install destinations. For each of them, the macro will create a helper variable `PACKAGE_<var...>`. These helper variables must be used in the `FooConfig.cmake.in` file for setting the installed location. They are calculated by `configure_package_config_file()` so that they are always relative to the installed location of the package. This works both for relative and also for absolute locations. For absolute locations, it works only if the absolute location is a subdirectory of `INSTALL_PREFIX`.

`DYNINST_INSTALL_INCLUDEDIR` is one of the [`PATH_VARS` passed to `configure_package_config_file()`](https://github.com/dyninst/dyninst/blob/effe9b22f88d984203821c6c0b3be43f7c0455c1/cmake/DyninstInstall.cmake#L14), so the variable was already available but not used.

Reply:
Ah. I should read more of the docs on confighelpers.